cmd/hvclient/flags.go

```diff
@@ -59,6 +59,7 @@ var (
 // Subject distinguished name flags.
 var (
 	fSubjectCommonName         = flag.String("commonname", "", "subject common name")
+	fSubjectSerialNumber       = flag.String("serialnumber", "", "subject serial number")
```

Comment:
Maybe worth adding something like "(distinct from certificate serial number)" as this confused me originally!

Reply:
Good idea!